### DISCUSSION TITLE
Can I install package globally

### DISCUSSION THREAD

**Discussion by zina0316** [python-poetry/poetry#8132] [Q&A]
I want to install package globally
but poetry says `Poetry could not find a pyproject.toml file`
so can I install package globally?

**Comment by radoering** [comment]
Poetry manages projects, not the global Python environment. That's why you need a pyproject.toml.

You can install your project's dependencies in the global environment, however that's not recommended. See #6398.

**Reply by zina0316** [reply]
then how to make a global environment?

**Reply by radoering** [reply]
https://python-poetry.org/docs/configuration/#virtualenvscreate
